### PR TITLE
[*] Segment - Handle association error on resource count 

### PR DESCRIPTION
**Pull request by ArnaudValensi**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request
- [ ] Write changes made in the CHANGELOG.md
- [ ] Create automatic tests
- [ ] No automatic tests failures
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by ArnaudValensi** [comment]
[P1 - Qonto - Accessing to a bad segment crash their server](https://trello.com/c/v2kokrlL/2929-p1-qonto-accessing-to-a-bad-segment-crash-their-server)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Segment - Handle filter on a bad association or a bad field in resource getter.
```
Comment (VincentMolinie) [inline CHANGELOG.md]:
```suggestion
- Collection - Avoid crash when filtering on a bad association or a bad field in resource getter.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+- Technical - Add test for resource getter.
```
Comment (VincentMolinie) [inline CHANGELOG.md]: Could you be more explicit here

File: app/services/forest_liana/operator_value_parser.rb [hunk 3]
```diff
@@ -93,6 +93,10 @@ def self.format_value(resource, field, value)
       field_name = field
       column_found = columns.find { |column| column.name == field_name }
 
+      if column_found.nil?
+        raise ForestLiana::Errors::HTTP404Error.new("Column not found")
```
Comment (VincentMolinie) [inline app/services/forest_liana/operator_value_parser.rb]: Should not we add the column name here ?

**Pull request merged**